plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/AbstractTestOracleTypeMapping.java

```diff
@@ -708,9 +709,10 @@ public void testUnsupportedDate()
             assertQueryFails(
                     format("INSERT INTO %s VALUES (DATE '0000-01-01')", table.getName()),
                     "\\QFailed to insert data: ORA-01841: (full) year must be between -4713 and +9999, and not be 0\n");
+            // The error message sounds invalid date format in the connector, but it's no problem as the max year is 9999 in Oracle
```

Comment:
Do we have a test to validate our assumption about max date in Oracle so that we know when Oracle behaviour changes?

Reply:
I'm not sure why you are asking this question because the following test L713-714 verifies the failure of Oracle max date + 1d.